sos/utilities.py

```diff
@@ -132,6 +132,29 @@ def convert_bytes(num_bytes):
     return f"{num_bytes}"
 
 
+def file_is_certificate(fname):
+    """Helper to determine if a given file is a certificate or not.
+
+    This is especially helpful for `sos clean`, which cannot obfuscate
+    certificate files content and instead, by default, will keep them as is.
+
+    :param fname:   The full path of the file to check
+    :type fname:    ``str``
+
+    :returns:   The type of the certificate or ``None``
+    :rtype:     ``string`` or ``None``
+    """
+    if fname[-4:] in [".csr", ".pem"]:
```

Comment:
Can we check against `.crt` suffix as well? E.g. various Satellite-related certificates (though afaik neither of them being collected by sos, in neither of plenty locations) have that suffix.

(tiny update: even sosreport from Satellite can contain `./etc/pki/pulp/qpid/ca.crt` until Qpid broker is deprecated in Sat)